vulkaninfo/vulkaninfo.h

```diff
@@ -1196,7 +973,7 @@ static VkSurfaceKHR AppCreateGgpSurface(AppInstance &inst) {
     createInfo.streamDescriptor = 1;
 
     VkSurfaceKHR surface;
-    VkResult err = inst.dll.fp_vkCreateStreamDescriptorSurfaceGGP(inst.instance, &createInfo, NULL, &surface);
+    VkResult err = vkCreateStreamDescriptorSurfaceGGP(inst.instance, &createInfo, NULL, &surface);
```

Comment:
This one seems unusual; I'm curious to know why we're supporting a `GGP` extension (is that Google?).

Reply:
Because it was added at some point. GGP means google games platform, AKA Stadia.